src/lib/edu-api.js

```diff
@@ -0,0 +1,15 @@
+import superagent from 'superagent';
+import superagentJsonapify from 'superagent-jsonapify';
+import apiClient from 'panoptes-client/lib/api-client';
+import { config } from './config';
+
+superagentJsonapify(superagent);
+
+export function get(endpoint) {
+  return superagent.get(`${config.root}${endpoint}`)
+    .set('Content-Type', 'application/json')
+    .set('Authorization', apiClient.headers.Authorization)
+    .then(response => response);
```

Comment:
I think returning `response` as is works better, actually. This would mean that whatever component/feature is calling the `get()` function will also be responsible for handling the error if, e.g. `response.statusCode === 404`.

If we want `get()` to return `response.body.data`, we'll need to bake error/bad response handling into this function, and it can get pretty unwieldy.